lib/libbackscrub.h

```diff
@@ -0,0 +1,54 @@
+/* This is licenced software, @see LICENSE file.
+ * Authors - @see AUTHORS file.
+==============================================================================*/
+
+// tested against tensorflow lite v2.4.1 (static library)
+#ifndef _LIBBACKSCRUB_H
+#define _LIBBACKSCRUB_H
+
+// for cv::Mat and related types
+#include <opencv2/core/core.hpp>
+#include <opencv2/imgproc/imgproc.hpp>
+#include <stdarg.h>
```

Comment:
I prefer to have system headers go before other dependencies and before headers from the own project. In this case:
* `libc`: System libs
* `opencv`, `tflite`: external dependencies
* `libvideoio`: internal dependencies
* `libbackscrub`: own library

Reply:
Agreed, I think I just missed this :smile: